src/split-panel/implementation.tsx

```diff
@@ -58,6 +58,10 @@ export const SplitPanelImplementation = React.forwardRef<HTMLElement, SplitPanel
     const openButtonAriaLabel = i18nStrings.openButtonAriaLabel;
     useEffect(() => {
       setSplitPanelToggle({ displayed: closeBehavior === 'collapse', ariaLabel: openButtonAriaLabel });
+
```

Comment:
Two questions:

1. Should the split panel state be reset upon reenabling?  Currently if you open drawer, then disable it, then re-enable it the split panel is showing in either the bottom and side position. Same for if you change the positioning. Is this what the cusomers expect? OR should the values be reset? 

2. Can we add an integ test for this? perhaps in the `app-layout-with-split-panel.test.tsx` or perhaps add its own test such as `runtime-split-panel-controllability.test.tsx` if applicable.  Here we start with it enabled, and disable it, and also enable it from it being disabled?

Reply:
1. Yes, I think this behavior is expected
2. I don't see much value adding an integ tests for that. But I agree that enabling split panel after disabling is worth adding to the existing u test